pkg/updater/service/helper/main.go

```diff
@@ -80,6 +80,11 @@ func buildCommand(inputCommand privilegeCommand) (*exec.Cmd, error) {
 		return exec.Command("mv", "/tmp/ld.so.preload.tmp", "/etc/ld.so.preload"), nil
 	case "add-installer-to-agent-group":
 		return exec.Command("usermod", "-aG", "dd-agent", "dd-installer"), nil
+	case "set-selinux-permissions":
```

Comment:
nice to get into it, a good pointer may be what's done for system-probe and security agent
- [posttrans](https://github.com/DataDog/datadog-agent/blob/main/omnibus/package-scripts/agent-rpm/posttrans#L38-L78)
- [system-probe policy](https://github.com/DataDog/datadog-agent/blob/main/cmd/agent/selinux/system_probe_policy.te)
if you don't find easily, I planned to work on SELinux next week